mimic/rest/maas_api.py

```diff
@@ -154,7 +161,12 @@ def create_entity(clock, params):
         ``bool``, ``dict`` or ``NoneType``.
     """
     current_time_milliseconds = int(1000 * clock.seconds())
-    params_copy = dict(params)
+    params_copy = _only_keys(params, ['agent_id',
```

Comment:
Non blocker: I feel like the keys list for each object should be in a different file, so one could see all the keys supported for each of these calls. I know from using the ele docs that this information was hard to find and having this here makes it lost in the midst of all this code.